### ISSUE TITLE
support annotation by ID

### ISSUE THREAD

**Issue by jjgao** [genome-nexus/genome-nexus#201]
Can we also support IDs under annotation-controller?

`/annotation/dbsnp/[dbsnp_id]`
`/annotation/cosmic/[cosmic_id]`
`/annotation/hgmd/[hgmd_id]`

We can utilize the following VEP endpoints:

https://rest.ensembl.org/documentation/info/vep_id_get
https://rest.ensembl.org/documentation/info/vep_id_post

related: https://github.com/genome-nexus/g2s/issues/52

**Comment by onursumer** [comment]
Seems like we may need to add `ColocatedVariant`s to our existing `VariantAnnotation` model to properly support these endpoints. Especially, the example cosmic endpoint doesn't even return transcript information, only colocated variants field is there.

Also, do you think we need to model all the fields for `ColocatedVariant`?

**Comment by jjgao** [comment]
@onursumer ok if we cannnot support COSMIC for now. Woud our current data model work for dbsnp?

**Comment by onursumer** [comment]
@jjgao I think we can still support both COSMIC and dbsnp, but only report the basic information not the colocated variants part for COSMIC. And, I think our current `Transcript` model should work for dbsnp annotation.

**Comment by jjgao** [comment]
@onursumer agreed. I don't think we need the `colocated_variants` part. If we need them, we could pull directly from our database.

**Comment by inodb** [comment]
FYI @emz1003 is working on this
